### DISCUSSION TITLE
tracing: I don't see any spans

### DISCUSSION THREAD

**Discussion by metanovii** [linkerd/linkerd2#5095] [General] [answered]
hi, i tried version stable-2.8.1 and edge-20.10.3:
```sh
linkerd2 install --addon-config config.yaml | kubectl apply -f -
```
config.yaml:
```yaml
tracing:
  enabled: true
```

and default annotations for deployments:
```
config.alpha.linkerd.io/trace-collector-service-account: linkerd-collector
config.linkerd.io/proxy-log-level: debug
config.linkerd.io/trace-collector: linkerd-collector.linkerd:55678
linkerd.io/inject: enabled
```

And what i see in jaeger:
![image](https://user-images.githubusercontent.com/9963998/96234522-3ac46480-0fa2-11eb-8c91-2a0459823714.png)

What am I doing wrong?

**Comment by grampelberg** [comment, marked as answer]
Have you followed the [tutorial](https://linkerd.io/2/tasks/distributed-tracing/)?

**Comment by metanovii** [comment]
Yes, I didn't read it carefully. However, tracing is possible without making changes to the code. :D

**Reply by grampelberg** [reply]
Tracing doesn't require code changes? You wouldn't happen to have an example I could check out? That'd be awesome!

**Reply by metanovii** [reply]
:D

**Reply by grampelberg** [reply]
I'll need to know more to help out, that's why I asked if you'd gone through the tutorial completely. If that's failing we can debug that. If that works but your app is failing, then we can debug that =)
